broke-broker/src/main/java/org/dsngroup/broke/broker/channel/handler/MqttMessageHandler.java

```diff
@@ -0,0 +1,67 @@
+package org.dsngroup.broke.broker.channel.handler;
+
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelInboundHandlerAdapter;
+import io.netty.handler.codec.mqtt.*;
+import org.dsngroup.broke.protocol.ConnectMessage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MqttMessageHandler extends ChannelInboundHandlerAdapter{
+
+    private final static Logger logger = LoggerFactory.getLogger(MqttMessageHandler.class);
+
+    private boolean isConnected = false;
+
+    @Override
+    public void channelRead(ChannelHandlerContext ctx, Object msg) {
+        MqttMessage mqttMessage = (MqttMessage) msg;
+        switch(mqttMessage.fixedHeader().messageType()) {
+            case CONNECT:
+                processConnect(ctx, (MqttConnectMessage)mqttMessage) ;
+                break;
+        }
+
+    }
+
+    @Override
+    public void channelActive(ChannelHandlerContext ctx) {
+        // Initialization of channel handler
+    }
+
+    /**
+     * The logic to deal with CONNECT message.
+     * @param ctx Channel Handler Context {@see ChannelHandlerContext}
+     * @param mqttConnectMessage Instance of MqttConnectMessage
+     * */
+    private void processConnect(ChannelHandlerContext ctx, MqttConnectMessage mqttConnectMessage) {
+
+        // Close the channel if receive the connect message second time.
+        if(this.isConnected) {
+            ctx.channel().close();
```

Comment:
The double connection message can be discarded, close it seems like not correct. Also, check this in another place, instead of directly check on this class.